### ISSUE TITLE
Is there a way to allow only certain users to trigger the release?

### ISSUE THREAD

**Issue by CermakM** [user-cont/release-bot#249]

**Comment by jpopelka** [comment]
No, not at the moment AFAIK, but shouldn't be hard to add such feature ([ogr.GithubProject.who_can_merge_pr()](https://github.com/packit-service/ogr/blob/master/ogr/services/github/project.py#L217))

Renaming issue.
